src/Logger/MiniLogger.cs

```diff
@@ -0,0 +1,107 @@
+using System;
+using Chorizo.Logger.Configuration;
+using Chorizo.Logger.Exceptions;
+using Chorizo.Logger.Output;
+using Chorizo.Logger.Output.Console;
+using Chorizo.Logger.Output.File;
+
+namespace Chorizo.Logger
+{
+    public class MiniLogger : IMiniLogger
```

Comment:
Can you refactor this so that it adheres to the open/closed principle? What if I want to write logs to somewhere other than _userInterfaceOut or _fileOut, or all, or none?